src/components/ResultContainer.tsx

```diff
@@ -84,13 +86,15 @@ function ResultContainer({ response }: { response: QueryResponse | null }) {
               [
                 res.dataset_uuid,
                 'protected', // subject_id
+                'protected', // session_id
+                'protected', // session_type
                 'protected', // age
                 'protected', // sex
                 'protected', // diagnosis
                 'protected', // assessment
-                'protected', // session_id
                 'protected', // session_file_path
-                'protected', // num_sessions
+                'protected', // num_phenotypic_sessions
+                'protected', // num_imaging_sessions
                 'protected', // image_modal
               ].join('\t')
```

Comment:
Just noticed that we should be able to do the "protected" bit here by using `repeatString('protected', headers.length-2)` or something like that? Maybe not worth the effort / less readable?

Reply:
Yea I think it would be less readable.